src/main/java/de/medizininformatik_initiative/process/report/service/CheckSearchBundle.java

```diff
@@ -32,14 +35,15 @@ public class CheckSearchBundle extends AbstractServiceDelegate
 	private static final String CAPABILITY_STATEMENT_PATH = "metadata";
 	private static final String SUMMARY_SEARCH_PARAM = "_summary";
 	private static final String SUMMARY_SEARCH_PARAM_VALUE_COUNT = "count";
+	private static final String TYPE_SEARCH_PARAM = "type";
 
 	private static final List<String> DATE_SEARCH_PARAMS = List.of("date", "recorded-date", "onset-date", "effective",
 			"effective-time", "authored", "collected", "issued", "period", "location-period", "occurrence");
-	private static final List<String> CODE_SEARCH_PARAMS = List.of("code", "ingredient-code");
-	private static final List<String> OTHER_SEARCH_PARAMS = List.of("_profile", "type", SUMMARY_SEARCH_PARAM);
+	private static final List<String> TOKEN_SEARCH_PARAMS = List.of("code", "ingredient-code", TYPE_SEARCH_PARAM);
```

Comment:
@wetret  is this not inconsistent to have some of the token params as strings and one via variable?

Reply:
The one via variable is used multiple times in this file, the others are not.

Reply:
I'm also not a big fan of such constants because they are an indirection and all such strings should be covered by tests anyway.